src/xrAICore/Navigation/level_graph.cpp

```diff
@@ -11,6 +11,7 @@
 #include "xrEngine/profiler.h"
 
 CLevelGraph::CLevelGraph(const char* fileName)
+    : m_level_id(GameGraph::_LEVEL_ID(-1))
```

Comment:
We can leave this change in this pull request.

Reply:
Well, done).

Reply:
@dimhotepus, but you didn't moved **other** changes to this file and `problem_solver_inline.h` to another pull request